### PR TITLE
Portfolio setup and mobile first

### PR DESCRIPTION
**Pull request by kambereBr**
In this pull request I:

1. Set up HTML & CSS linter.
2. Built a personal portfolio site that contains 2 sections (the toolbar and the headline section)
3. Made the website responsive for both mobile screen and desktop screen.
4. Added README.md file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        </nav>
+    </header>
+    <section class="main-section">
+        <div class="div-description">
+            <h2>I'm Bruno.<br />Glad to see you!</h1>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        </div>
+        <div class="div-connect">
+            <p>LET’S CONNECT</p>
```
Comment (bienvenuushindi) [inline index.html:31]: :red_circle: I think this looks more like a title than a paragraph. It should use a header tag hx (examples: `<h4>,<h3>,<h2>,` etc...). What do you think :thinking:

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
### Hi @kambereBr      ,

Good job implementing most of the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (kambereBr) [reply index.html]: You're right, thanks @bienvenuushindi

**Review by shella12 — APPROVED** [review]
Hi @kambereBr, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Clean Code ✨ 
- Correct use of semantic tags 👍 
- Implemented all requirements 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**